codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/customize/CoreCodegenDecorator.kt

```diff
@@ -63,6 +66,31 @@ interface CoreCodegenDecorator<CodegenContext> {
         baseCustomizations: List<LibRsCustomization>,
     ): List<LibRsCustomization> = baseCustomizations
 
+    /**
+     * Hook to customize generated structures.
+     */
```

Comment:
Are these structures written to a specific module? What separates this from `extras`? It'd be nice to have a "map" of where all this stuff ends up.

Reply:
These are customizations for the `StructureGenerator`. I'll clarify the comment.